src/ui/server/helpers/is-populated-array/index.test.ts

```diff
@@ -0,0 +1,29 @@
+import isPopulatedArray from '.';
+
+describe('server/helpers/is-populated-array', () => {
+  describe('when an array is populated', () => {
+    it('should return true', () => {
+      const mockArray = [{ workId: 1 }, { workId: 2 }];
+
+      const result = isPopulatedArray(mockArray);
+
+      expect(result).toEqual(true);
+    });
+  });
+
+  describe('when an array is not provided', () => {
+    it('should return false', () => {
+      const result = isPopulatedArray();
+
+      expect(result).toEqual(false);
+    });
+  });
+
+  describe('when the array is empty', () => {
+    it('should return false', () => {
```

Comment:
Can we add a test case passing an object with few filled properties please.

Reply:
Had a chat, pushed a test for scenario where mulesoft could pass an object to this function.